### PR TITLE
Review installation using pip

### PR DESCRIPTION
**Pull request by jburel**
see https://github.com/ome/omero-documentation/issues/2112 for background
* remove usage of upgrade
*  upgrade pip

### CODE REVIEW

File: ansible/templates/common/install.sh.j2 [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+    /opt/omero/web//venv3/bin/pip install --upgrade pip
+    /opt/omero/web//venv3/bin/pip install "omero-web>=\ |version_web|"
+
 {% else %}
 #!/bin/bash
 
 set -e -u -x
 
+{{ virtualenv_path }}/bin/pip install --upgrade pip
+{{ virtualenv_path }}/bin/pip install "omero-web>=5.6.1"
```
Comment (sbesson) [inline ansible/templates/common/install.sh.j2:15]: if this is an installation from scratch, is there a reason to specify a minimum version or can it be omitted as `pip install` should default to the latest version satisfying the package constraints?

File: ansible/templates/common/install.sh.j2 [hunk 2]
```diff
@@ -1,11 +1,17 @@
 {% if doc %}
-Install OMERO.web::
+Upgrade pip and install OMERO.web:
+
+.. parsed-literal::
+
+    /opt/omero/web//venv3/bin/pip install --upgrade pip
+    /opt/omero/web//venv3/bin/pip install "omero-web>=\ |version_web|"
```
Comment (sbesson) [inline ansible/templates/common/install.sh.j2:7]:
```suggestion
    /opt/omero/web//venv3/bin/pip install omero-web
```

Comment (jburel) [reply ansible/templates/common/install.sh.j2:7]: This does not match what we have put in place in the doc for other packages e.g. dropbox

Comment (sbesson) [reply ansible/templates/common/install.sh.j2:7]: I understood this block was meant to be identical to https://github.com/ome/omeroweb-install/pull/77/commits/bf0e96566c4d21f68ec15902a84bfedd0992db69 with a different venv prefix?

No objection to keeping it as it is but for me the same question as https://github.com/ome/omeroweb-install/pull/77#discussion_r764128654 applies. Where is the dropbox line you are referring to?

Comment (jburel) [reply ansible/templates/common/install.sh.j2:7]: See https://github.com/ome/omero-documentation/pull/2109
To be honest, dropping the version makes this easier i.e. no need for ``if else``

Comment (jburel) [reply ansible/templates/common/install.sh.j2]: I have pushed a commit dropping the version

**Review by sbesson — APPROVED** [review]

**Pull request merged**